core/assets/bundles/bundle_de.properties

```diff
@@ -760,8 +760,8 @@ keybind.toggle_block_status.name = Toggle Block Statuses
 keybind.move_x.name = X-Achse
 keybind.move_y.name = Y-Achse
 keybind.mouse_move.name = Der Maus folgen
-keybind.pan.name = Pan View
-keybind.boost.name = Boost
```

Comment:
Boost lassen bzw. wenn dann in "Versätrkung" umbenennen, "Flug" ist hier falsch

Reply:
Boost lassen.